tests/e2e/testutils.go

```diff
@@ -437,16 +443,19 @@ func RestartClusterAgent(nodeNames []string) error {
 
 // RunCmdOnNode executes a command from within the given node
 func RunCmdOnNode(cmd string, nodename string) (string, error) {
-	runcmd := "vagrant ssh -c \"" + cmd + "\" " + nodename
+	runcmd := "vagrant ssh " + nodename + " -c \"" + cmd + "\""
 	out, err := RunCommand(runcmd)
 	if err != nil {
-		return out, fmt.Errorf("failed to run command %s on node %s: %v", cmd, nodename, err)
+		return out, fmt.Errorf("failed to run command: %s on node %s: %s, %v", cmd, nodename, out, err)
 	}
 	return out, nil
 }
 
-// RunCommand executes a command on the host
 func RunCommand(cmd string) (string, error) {
+	// we don't use c.env because we embed the command inside a bash shell
```

Comment:
I'm a little confused by this statement; any environment variable you set in `c.Env` should also be available to things executing in the shell, as subprocesses of it.

Reply:
I was running into issue around using `c.env` and bash -c vagrant -c was getting called. I'm still tinkering.